dglgo/setup.py

```diff
@@ -16,7 +16,10 @@
           'numpydoc>=1.1.0',
           "pydantic>=1.9.0",
           "ruamel.yaml>=0.17.20",
-          "PyYAML>=5.1"
+          "PyYAML>=5.1",
+          "ogb>=1.3.3",
+          "rdkit-pypi",
```

Comment:
Is it very difficult to install `rdkit`? If so, suggest to remove it and make related import optional.

Reply:
Someone built a pip distribution for `rdkit`, so it's no longer the case.